sagemaker-debugger/tensorflow_action_on_rule/detect_stalled_training_job_and_stop.ipynb

```diff
@@ -4,30 +4,22 @@
    "cell_type": "markdown",
```

Comment:
<p>What if customer is not using default container?</p><p>We should have a section explaining that customers can do hook.save_scalar periodically in their training/validation loop as heartbeat mechanism.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1592/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Added the hook registration code lines as comments in the training script. Also mentioned about it in the notebook.